composer.json

```diff
@@ -110,19 +110,18 @@
     },
 
     "require-dev": {
-        "doctrine/doctrine-bundle":     "^2.0",
-        "doctrine/orm":                 "^2.6.4",
-        "symfony/browser-kit":          "^4.4|^5.1",
+        "doctrine/doctrine-bundle":     "^2.4",
+        "doctrine/orm":                 "^2.9",
+        "symfony/browser-kit":          "^4.4.20|^5.1",
         "symfony/css-selector":         "^4.4|^5.1",
         "symfony/property-access":      "^4.4|^5.1",
         "symfony/validator":            "^4.4|^5.1",
         "symfony/twig-bundle":          "^4.4|^5.1",
         "symfony/stopwatch":            "^5.1",
         "symfony/translation":          "^4.4|^5.1",
-        "friendsofsymfony/user-bundle": "^2.1",
         "php-http/httplug-bundle":      "^1.7",
         "php-http/guzzle6-adapter":     "^2.0",
-        "phpunit/phpunit":              "^9.5",
+        "phpunit/phpunit":              "9.5.5",
```

Comment:
Why is this needed?

Reply:
Some left-over during debug probably, reverted.